lambda/interceptors.js

```
@@ -36,7 +37,7 @@ const AddDirectiveResponseInterceptor = {
         )
       : false;
     console.log("APL Directive: ", JSON.stringify(aplDirective));      
-    const ssmlText = response.outputSpeech.ssml;      
+    const ssmlText = response && response.outputSpeech && response.outputSpeech.ssml ? response.outputSpeech.ssml : null;      
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)